### ISSUE TITLE
Purge folder before upload

### ISSUE THREAD

**Issue by Liingon** [banago/PHPloy#364]
I need to purge a folder before uploading. Is there a way to do this currently? From what I understand purge is called after the upload of new files is complete.

**Comment by pashkasan** [comment]
pre-deploy-remote[] = "rm -rf folder/*" ?

**Comment by Liingon** [comment]
Yeah, should have mentioned that I don't have ssh access to the server so I need something that works over ftp.
